python_bindings/src/halide/halide_/PyHalide.h

```diff
@@ -37,7 +37,57 @@ Expr double_to_expr_check(double v);
 Target to_jit_target(const Target &target);
 Target to_aot_target(const Target &target);
 
+// TODO: when we move to C++20 & our base toolchains are modern enough,
```

Comment:
Can you create an issue for this to track, and add it to the comment?  I think that will make it easier for us to do this when we actually move to C++20